### ISSUE TITLE
[mvt.android.modules.adb.chrome_history] Running module ChromeHistory...

### ISSUE THREAD

**Issue by g1rafucker** [mvt-project/mvt#396]
stuck on [mvt.android.modules.adb.chrome_history] Running module
ChromeHistory...
any clues? its been 30mins

**Comment by Te-k** [comment]
Hi,

Is your phone rooted?

**Comment by electromage** [comment]
I've hit this also. My phone is not rooted - is that a requirement? Is there a way around this? My understanding is that rooting the phone can prevent new security issues.

**Comment by Te-k** [comment]
Yes, MVT can only access Chrome history if your phone is rooted, which definitely creates some security issues. That said, it should tell you it couldn't access and not block on it. I am not sure what is the issue there.
What is your android version? Can you check if you can access `/data/data/com.android.chrome/app_chrome/Default/History` in adb?

**Comment by lost-in-bits** [comment]
> Yes, MVT can only access Chrome history if your phone is rooted, which definitely creates some security issues. That said, it should tell you it couldn't access and not block on it. I am not sure what is the issue there. What is your android version? Can you check if you can access `/data/data/com.android.chrome/app_chrome/Default/History` in adb?

I'm having the same issue. when I run adb shell getprop ro.build.version.release, adb returns "13". 

Additionally, I get "adb: inaccessible or not found" when I try to access /data/data/com.android.chrome/app_chrome/Default/History.

Let me know if you need any more information.

**Comment by Te-k** [comment]
So the modules run and it gets stuck?

**Comment by Joedmin** [comment]
Is it possible to skip this check on a non-rooted device?

**Comment by oefielixir** [comment]
Any news on that?

**Comment by dsh0** [comment]
Until the issue is fixed, when ADB ask to authorise the connected computer, you need to check the box to remember the device. ChromeHistory hangs if you give one time permission, but if the box is checked it detects immediately that the device is not rooted and moves on. You can always revoke authorisation later.

**Comment by Nader-B10** [comment]
![Screenshot 2024-08-17 173912](https://github.com/user-attachments/assets/916d9f90-80b4-4ea7-8989-b3fc488141b4)
how to fix this?

**Comment by Benjipeper** [comment]
hey guys a got a really weird problem, i got my android listed as adb devices through "adb connect [IP-adress]:5555"
and try to run the "mvt-android check-adb" its start running but then still gives the "No devices found" after running "chrome_history" any ideas what might causes this?

**Comment by besendorf** [comment]
ADB functionality has been removed from MVT please use https://github.com/mvt-project/androidqf going forward
